project/ws/app/src/lib/routes/home/routes/designation/components/bulk-upload/bulk-upload.component.html

```diff
@@ -0,0 +1,109 @@
+<div class="flex flex-col gap-6">
+  <div class="flex flex-row gap-4 items-center">
+    <mat-icon class="cursor-pointer" [routerLink]="['/app/home/org-designations']">arrow_back</mat-icon>
+    <div class="flex font-bold">Bulk Upload Designations</div>
+  </div>
+
+  <div class="flex flex-col gap-4 bulk-container">
+    <div class="flex font-bold">Bulk Upload Designations</div>
+    <div class="flex flex-row gap-8">
+      <div class="flex-1">
+        <div class="flex flex-col gap-3 dashed-card">
+          <div>Follow these steps to update multiple user profiles:</div>
```

Comment:
Add all these texts from json file in client assets

Reply:
Added text from config